homeworks/denys.viitenko_danviitenko/homework_6/MA-github-client/MA-github-client/Search/SearchViewController.swift

```diff
@@ -0,0 +1,75 @@
+//
+//  SearchViewController.swift
+//  MA-github-client
+//
+//  Created by Dan on 12/17/18.
+//  Copyright © 2018 Dan Viitenko. All rights reserved.
+//
+
+import UIKit
+
+class SearchViewController: UIViewController, UITableViewDelegate,UITableViewDataSource,UISearchBarDelegate {
+    var repositories = [Repo]()
+    @IBOutlet weak var searchBar: UISearchBar!
+    @IBOutlet weak var tblView: UITableView!
+    
+    var dataFiltered: [Repo]!
```

Comment:
Лучше исключать force unwrap-а.
```suggestion
    var dataFiltered = [Repo]()
```